### DISCUSSION TITLE
Create JPEG Thumbnail not working

### DISCUSSION THREAD

**Discussion by JamieLee2k** [mriscoc/Ender3V2S1#1068] [General]
I have an Ender 3 V2 and I am unable to get this preview working, I have updated the firmware, the screen and even put the file in the script folder within Cura, I then goto Post Processing Scripts - Modify Gcode and add Create JPEG Thumbnail as a script leaving everything default, its not showing up, what did I do wrong, oh and yes I did restart Cura twice.

I have the Dacai Screen flashed with the files suggested by Mriscoc firmware.

**Comment by mriscoc** [comment]
https://github.com/mriscoc/Ender3V2S1/wiki/How-to-generate-a-gcode-preview#troubleshooting

**Comment by zaggash** [comment]
> I have an Ender 3 V2 and I am unable to get this preview working, I have updated the firmware, the screen and even put the file in the script folder within Cura, I then goto Post Processing Scripts - Modify Gcode and add Create JPEG Thumbnail as a script leaving everything default, its not showing up, what did I do wrong, oh and yes I did restart Cura twice.
> 
> I have the Dacai Screen flashed with the files suggested by Mriscoc firmware.

I had the same issue.
Be careful, there is a default Post Processing Scripts from Cura called: ***Create Thumbnail***

The one you copy from this repo is called: ***Create JPEG Thumbnail***

So be sure you selected the latter in your Post Processing Scripts !

**Reply by JamieLee2k** [reply]
I have the right script, the only thing it could be is the display firmware, I used the one that changes the theme to look nicer, it has a splashscreen of Mriscoc so it should be included.

**Reply by zaggash** [reply]
Did you test with the gcode logo example file?

**Reply by JamieLee2k** [reply]
It shouldn't matter what file I use, I used a Benchy but nothing is showing up

**Reply by zaggash** [reply]
Indeed it doesn't matter.
What matter is to check if the problem comes from the display (printer) or the converter (cura)

The example file is working, so if nothing is showing up, the problem comes from the display.

If it does show up on the display, then the problem comes from the converter.

**Reply by JamieLee2k** [reply]
I did update the display but only with a theme one rather than the official one, I can see that might be the issue but I am sure Mriscoc has suggested that theme based on the splashscreen says Mriscoc but as for why the preview isn't included in the code is a mystery

**Reply by zaggash** [reply]
But are you sure it is not included?!?
You make assertions without conducting a proper test.
This will be my last message tho.
